tests/unit/test_wloader.py

```diff
@@ -165,15 +165,16 @@ def testDumpUnicodeSupport(self):
         node = next(iter(self.loader.loads(text)))
         dump = node.dump()
         self.assertTrue(isinstance(dump, six.text_type))
-        self.assertEquals(dump, text)
+        self.assertEqual(dump, text)
 
     def testStrUnicodeBuiltIns(self):
         node = next(iter(self.loader.loads("\n option value")))
         self.assertTrue(isinstance(str(node), str))
         self.assertTrue(isinstance(node.__unicode__(), six.text_type))
-        self.assertEquals(
-            "LeafNode([u'statement', u'option', u' ', u'value'])",
-            six.text_type(node))
+        node_str = six.text_type(node).replace("u'", "'")
+        self.assertEqual(
+            "LeafNode(['statement', 'option', ' ', 'value'])",
+            node_str)
```

Comment:
This is to make the test compatible with both `u'string'` and `'string'`, depending on whether it was being tested in Python2 or Python3.

Reply:
Arguably, explicit branching on Python version might look less confusing...?